src/modules/75-cf/pages/feature-flags/components/StaleFlagActions/StaleFlagActions.tsx

```diff
@@ -5,32 +5,82 @@
 import { useGetSelectedStaleFlags } from '../../hooks/useGetSelectedStaleFlags'
+import css from './StaleFlagActions.module.scss'
 
 export const StaleFlagActions: FC = () => {
   const { getString } = useStrings()
   const selectedFlags = useGetSelectedStaleFlags()
+  const [showInfo, setShowInfo] = useState<boolean>(false)
+
+  const hideShowInfo = useCallback(() => {
+    setShowInfo(false)
+  }, [])
 
   return (
     <>
       {!!selectedFlags.length && (
-        <Layout.Horizontal padding="xlarge" border={{ top: true }} background={Color.WHITE} spacing="medium">
-          <Text font={{ variation: FontVariation.CARD_TITLE }}>
-            {getString('cf.staleFlagAction.flagsSelected', { count: selectedFlags.length })}
-          </Text>
-          <Button text={getString('cf.staleFlagAction.notStale')} variation={ButtonVariation.SECONDARY} />
-          <Button text={getString('cf.staleFlagAction.readyForCleanup')} variation={ButtonVariation.SECONDARY} />
-          {/* TODO: Add info popup for button below */}
-          <Button
-            padding={{ left: 0 }}
-            text={getString('cf.staleFlagAction.learnMore')}
-            variation={ButtonVariation.LINK}
-          />
-        </Layout.Horizontal>
+        <>
+          <Layout.Horizontal padding="xlarge" border={{ top: true }} background={Color.WHITE} spacing="medium">
+            <Text font={{ variation: FontVariation.CARD_TITLE }}>
+              {getString('cf.staleFlagAction.flagsSelected', { count: selectedFlags.length })}
+            </Text>
+            <Button text={getString('cf.staleFlagAction.notStale')} variation={ButtonVariation.SECONDARY} />
+            <Button text={getString('cf.staleFlagAction.readyForCleanup')} variation={ButtonVariation.SECONDARY} />
+            <Button
+              padding={{ left: 0 }}
+              text={getString('cf.staleFlagAction.learnMore')}
+              variation={ButtonVariation.LINK}
+              onClick={() => {
+                setShowInfo(true)
+              }}
+            />
+          </Layout.Horizontal>
+          {showInfo && (
+            <Drawer
+              onClose={hideShowInfo}
+              usePortal={true}
+              autoFocus={true}
+              canEscapeKeyClose={true}
+              canOutsideClickClose={true}
+              enforceFocus={false}
+              hasBackdrop={true}
+              isOpen={true}
+              position={Position.RIGHT}
+              size={'700px'}
+            >
+              <Button
+                className={css.closeButton}
+                icon="main-close"
+                iconProps={{ size: 12 }}
+                onClick={hideShowInfo}
+                aria-label="close"
+              />
+              <Layout.Vertical className={css.infoDrawer}>
+                <Text className={css.infoTitle} font={{ variation: FontVariation.H4 }}>
+                  {getString('cf.staleFlagAction.flagLifecycleExplained')}
+                </Text>
```

Comment:
Since this is a title, can we use the `Heading` component instead.
```suggestion
                <Header level={4} className={css.infoTitle} font={{ variation: FontVariation.H4 }}>
                  {getString('cf.staleFlagAction.flagLifecycleExplained')}
                </Header>
```

Reply:
Nice, thanks have updated